### PR TITLE
Contact_form

### PR DESCRIPTION
**Pull request by Sirvelasque**
-add contact section
-supporting text 
-contact form including inputs, textarea and submit button

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -133,6 +134,46 @@ <h3 class="works_titles">Multi-Post Stories</h3>
...
+          <ul>
+            <li>
+              <input type="text" id="name" name="Name" placeholder="Full name" maxlength="30" required>
+            </li>
+            <li>
+              <input type="email" id="email" name="E-mail" placeholder="Email" required>
+            </li>
+            <li>
+              <textarea id="message" cols="40" rows="10" maxlength="500"></textarea>
+            </li>
```
Comment (ShoiraTa) [inline html/index.html]: - [ ] Please kindly make the message input field `required` as well

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html/index.html [hunk 2]
```diff
@@ -133,6 +134,46 @@ <h3 class="works_titles">Multi-Post Stories</h3>
...
+        <form action="https://formspree.io/f/xqknednn" method="post">
+          <ul>
+            <li>
+              <input type="text" id="name" name="Name" placeholder="Full name" maxlength="30" required>
+            </li>
+            <li>
+              <input type="email" id="email" name="E-mail" placeholder="Email" required>
+            </li>
+            <li>
+              <textarea id="message" cols="40" rows="10" maxlength="500" required></textarea>
```
Comment (mateo951) [inline html/index.html:157]: Addressed 👌😎🔥

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Great work making the necessary changes requested by the previous reviewer 👌
Your project is complete!  There is nothing else to say other than... it's time to merge it 🧐 
### Congratulations! 🎉🤯🚀🔥


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**